jhotdraw-samples/jhotdraw-samples-misc/src/test/java/org/jhotdraw/samples/svg/figures/TextAreaGivenTest.java

```diff
@@ -0,0 +1,18 @@
+package org.jhotdraw.samples.svg.figures;
+
+import com.tngtech.jgiven.junit.ScenarioTest;
+
+import org.jhotdraw.samples.svg.figures.JGivenTextArea.GivenUserSelectTextArea;
+import org.jhotdraw.samples.svg.figures.JGivenTextArea.ThenTheTextShouldBeAdded;
+import org.jhotdraw.samples.svg.figures.JGivenTextArea.WhenUserWritesAFewLinesOfText;
+import org.junit.Test;
+
+public class TextAreaGivenTest extends ScenarioTest<GivenUserSelectTextArea, WhenUserWritesAFewLinesOfText, ThenTheTextShouldBeAdded> {
+
+    @Test
+    public void text_area() {
+        given().the_user_select_text_area();
```

Comment:
The methods does not follow the same naming conventions as those already in develop and even here :P
See the suggestions I provided.